Feliz/Styles.fs

```diff
@@ -77,6 +81,7 @@ type length =
     static member inline percent(value: int) : ICssUnit = unbox ((unbox<string>value) + "%")
     /// Relative to the parent element
     static member inline percent(value: double) : ICssUnit = unbox ((unbox<string>value) + "%")
+    /// The browser calculates the width.
```

Comment:
Shouldn't this be `The browser calculates the length`? beucase `ICssUnit` can be applied to more than just widths AFAIK